newscoop/application/configs/parameters/parameters.yml

```diff
@@ -134,3 +134,4 @@ parameters:
         environment: 'prod'
     simple_security:
         users_file: "%application_path%/configs/security/users.json"
+    remember_me.lifetime: 31536000 # 1 year in seconds
```

Comment:
Agree. Let's go with default value in `parameters.yml` and possibility to override it with `custom_parameters.yml`